build.sbt

```diff
@@ -80,6 +81,10 @@ val emailDependencies = Seq(
   "com.sun.mail" % "javax.mail" % "1.6.2" exclude ("javax.activation", "activation")
 )
 
+val macwireDependencies = Seq(
+  "com.softwaremill.macwire" %% "macrosautocats" % macwireVersion
```

Comment:
side-note for macwire - I think we should rename the module to sth readable, such as `macros-auto-cats` ;)